### ISSUE TITLE
ORCA dipole moment print outs?

### ISSUE THREAD

**Issue by mim134** [grimme-lab/xtb#1124]
I'm using xtb2 through ORCA 5.0. I can't get the dipole moment to print in the output file. It is not located in any of the scratch files either. 

My input and output file are attached. 
[xtb2_input.txt](https://github.com/user-attachments/files/17607759/xtb2_input.txt)
[xtb2_output.txt](https://github.com/user-attachments/files/17607760/xtb2_output.txt)

The output file also lists all the files in the scratch space at the end of the calculation.  I'm unsure of if there's a keyword I'm missing. I can't seem to find anything in the documentation.
Thanks!

**Comment by awvwgk** [comment]
This seems like something that needs to be handled on the Orca side rather than in xtb. For Orca to printout the xtb computed dipole moment, there needs to be an Orca xtb interface for this in the first place. To the best of my knowledge this does not exist in Orca yet.

**Comment by marcelmbn** [comment]
> This seems like something that needs to be handled on the Orca side rather than in xtb. For Orca to printout the xtb computed dipole moment, there needs to be an Orca xtb interface for this in the first place. To the best of my knowledge this does not exist in Orca yet.

Agree. The place to discuss such a feature would be the ORCA forum: https://orcaforum.kofo.mpg.de/index.php
